api/recover.go

```diff
@@ -26,13 +28,26 @@ func (a *API) Recover(w http.ResponseWriter, r *http.Request) error {
 		return badRequestError("Could not read verification params: %v", err)
 	}
 
-	if params.Email == "" {
-		return unprocessableEntityError("Password recovery requires an email")
+	if params.Email == "" && params.Phone == "" {
+		return unprocessableEntityError("Password recovery requires an email or a phone number")
 	}
 
+	var user *models.User
 	aud := a.requestAud(ctx, r)
-	user, err := models.FindUserByEmailAndAudience(a.db, instanceID, params.Email, aud)
 	recoverErrorMessage := "If a user exists, you will receive an email with instructions on how to reset your password."
+	if params.Email != "" {
```

Comment:
Ah ok so looks like we default to email if both email and phone are present?

Reply:
Yup we'll always prefer email for now since its more secure